### PR TITLE
added drawer menu with topics navigation

### PR DESCRIPTION
**Pull request by skelbon**

### CODE REVIEW

**Review by RichardNeat — COMMENTED** [review]
Looks really nice! 🎉 
One little thought, it would be nice if I when I opened up the comments for the article the articles on the other column didn't shift as well like they are static.

**Comment by skelbon** [comment]
Thanks Rich, originally I had the grid just one article wide. Not sure how difficult it will be to do as you suggest, I'll take a look at it - perhaps it would be better to revert to a single column in the meantime. It's supposed to be 'mobile first' anyway???

**Comment by RichardNeat** [comment]
Yeah that's legit for now to consider it primarily in mobile view 👍

File: src/components/SortBar.jsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    
+    const handleSortChange = (e)=>{
+        setSortBy(e.target.value)
+    }
+    
+    return (
+        <>
+        <Stack 
+            direction="row"
+            // spacing={4}
```
Comment (B7Bprog) [inline src/components/SortBar.jsx:29]: Please remove all commented-out code.

File: src/network/network.jsx [hunk 2]
```diff
@@ -1,7 +1,10 @@
 import axios from "axios";
 
-export const getArticles = async ()=>{
-    const articles = await axios.get("https://skelbon-news-api.onrender.com/api/articles")
+export const getArticles = async (filter, sortBy, order)=>{
+    const topic = filter ? ('?topic=' + filter + '&') : '?'
+ 
+    console.log((`https://skelbon-news-api.onrender.com/api/articles${topic}sort_by=${sortBy}&order=${order}`))
```
Comment (B7Bprog) [inline src/network/network.jsx:6]: Please remove all logs from this final code.

**Review by B7Bprog — COMMENTED** [review]
Hey Joe, nice work on sorting articles. Currently sorting by comment count doesn't work. Please fix that and check my other comments.

**Pull request merged**